### PR TITLE
Flow garden test

### PR DESCRIPTION
**Pull request by jlin703**
Flow garden testing script.

### CODE REVIEW

**Comment by eugenevinitsky** [comment]
@AboudyKreidieh we will simultaneously write a test for this and visualizer_rllib; I will set that up.

File: flow/benchmarks/test_bm.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+        # terminate the environment
+        env.terminate()
+
+        result_file = open(solution_dir + "/results.txt", "w")
+        result_file.write(str(np.mean(rets).round(4)) + '\n')
+        result_file.write(','.join([str(s) for s in rets]))
+
+    except Exception as e:
+        result_file.write(str(e))
```
Comment (kanaadp) [inline flow/benchmarks/test_bm.py]: nit: does this line overwrite "FAILED"  — if possible we should just append

File: flow/benchmarks/test_bm.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+            for _ in range(env_params.horizon):
+                action = compute_action(state)
+                state, reward, done, _ = env.step(action)
+                ret += reward
+                if done:
+                    break
+            rets.append(round(ret, 2))
+            print("Reward:", round(ret, 2))
+        print("Average, std return: {}, {}".format(
+            np.mean(rets), np.std(rets)))
```
Comment (kanaadp) [inline flow/benchmarks/test_bm.py]: NIT: indenting here

**Comment by AboudyKreidieh** [comment]
@kanaadp @eugenevinitsky do you guys still want to merge this?

**Comment by AboudyKreidieh** [comment]
going to close for now. if we want to merge this feel free to reopen

**Pull request closed without merging**